.github/workflows/unittests.yml

```diff
@@ -10,5 +10,8 @@ jobs:
       - name: Checkout GitHub workspace
         uses: actions/checkout@v2
 
-      - name: Install python, activate venv, install dependencies, & run tests all in docker container
-        run: sudo docker run --rm -v $PWD:/src -w "/src" ubuntu:latest bash -c "apt update && apt -y upgrade && apt install -y python3-pip python3-venv && ./setup.sh && source ./venv/bin/activate && ./runtests.sh"
+      - name: Install python, activate venv, install dependencies, & run tests
+        run: ./setup.sh
+        run: . venv/bin/activate
+        run: ./runtests.sh
+        run: deactivate
```

Comment:
There's a syntax error in line 14-17. See https://github.com/tetrabiodistributed/project-tetra-display/actions/runs/198822222
Try,
```
run: |
  ./setup.sh
  . venv/bin/activate
  ./runtests.sh
  deactivate
```

Reply:
also, since there's a `deactivate` added in `runtests.sh`, would it be redundant to also add it here?

Reply:
Fair point yeah